av_camera_launch/config/fsp_l_config.yaml

```
@@ -1,36 +1,47 @@
 /**:
   ros__parameters:
-    camera_name: "fsp_l"
-    camera_type: "chameleon"
-    serial_number: "17496155"
-    frame_id: "camera_fsp_l_optical"
-    parameter_file: "/opt/ros_ws/install/av_camera_launch/share/av_camera_launch/config/chameleon.yaml"
-    camerainfo_url: "file:///opt/ros_ws/install/av_camera_launch/share/av_camera_launch/config/fsp_l_calib.yaml"
-    debug: False
-    compute_brightness: False
-    dump_node_map: False
-    adjust_timestamp: True
-    gain_auto: "Continuous"
-    exposure_auto: "Off"
-    exposure_time: 1000
-    reverse_x: False
-    reverse_y: False
-    offset_x: 24
-    offset_y: 326
-    image_width: 2400
-    image_height: 1200
-    pixel_format: "BayerRG8"  # 'BayerRG8 'RGB8' or 'Mono8'
-    trigger_selector: "FrameStart"
-    trigger_mode: "On"
-    trigger_source: "Line3"
-    trigger_delay: 0
-    trigger_overlap: "ReadOut"
-    chunk_mode_active: True
-    chunk_selector_frame_id: "FrameID"
-    chunk_enable_frame_id: True
-    chunk_selector_exposure_time: "ExposureTime"
-    chunk_enable_exposure_time: True
-    chunk_selector_gain: "Gain"
-    chunk_enable_gain: True
-    chunk_selector_timestamp: "Timestamp"
-    chunk_enable_timestamp: True
+    fsp_l:
+      camera_name: "fsp_l"
+      camera_type: "chameleon"
+      serial_number: "17496155"
+      frame_id: "camera_fsp_l_optical"
+      parameter_file: "/opt/ros_ws/install/av_camera_launch/share/av_camera_launch/config/chameleon.yaml"
+      camerainfo_url: "file:///opt/ros_ws/install/av_camera_launch/share/av_camera_launch/config/fsp_l_calib.yaml"
+      debug: False
+      compute_brightness: True
+      dump_node_map: False
+      adjust_timestamp: True
+      gain_auto: "Continuous"
+      exposure_auto: "Off"
+      exposure_controller_name: "fsp_l.exposure_controller"
+      exposure_controller:
+        brightness_target: 120
+        brightness_tolerance: 20
+        gain_priority: false
+        max_exposure_time: 1000
+        max_frames_skip: 10
+        max_gain: 10.0
+        min_exposure_time: 1000
+        type: "master"
+      exposure_time: 5000
```

I think we want this at `1ms` ? 
```suggestion
      exposure_time: 1000
```

---

Fixed hehe